internal/service/wafv2/web_acl_data_source.go

```diff
@@ -32,12 +32,18 @@ func DataSourceWebACL() *schema.Resource {
 				},
 				"name": {
 					Type:     schema.TypeString,
-					Required: true,
+					Computed: true,
+					Optional: true,
+				},
+				"resource_arn": {
+					Type:     schema.TypeString,
+					Computed: true,
+					Optional: true,
 				},
 				"scope": {
-					Type:         schema.TypeString,
-					Required:     true,
-					ValidateFunc: validation.StringInSlice(wafv2.Scope_Values(), false),
+					Type:     schema.TypeString,
+					Computed: true,
+					Optional: true,
```

Comment:
I think this should stay as required

Reply:
It's checked later. If resource_arn isn't there this becomes required

Reply:
The scope should be required as this is defining what route the check will go, REGIONAL check is using one API call, and CLOUDFRONT another.

Reply:
Scope isn't used in the api call though.

Reply:
Ah nevermind, I though the resource arn would take Cloudfront as an input, but that's a different call.